motor_control/throttle/variable_control.ino

```diff
@@ -0,0 +1,22 @@
+#include "variable_control.h"
+
+VariableThrottle::VariableThrottle(const int &pin):Throttle(pin){}
+
+void VariableThrottle::compute_motor_value() {
+  /*
+   * Reads the throttle input from analog signal,
+   * Returns 0~1023
+   */
+  read_sensor_value();
+  int throttleReading = get_raw_value();
```

Comment:
You should be able to access `m_raw_value` directly here instead of using the getter function. So there's no need to declare this other variable `throttleReading`.